niap-cc/Permissions/Tester/app/src/main/java/com/android/certifications/niap/permissions/config/DevicePolicyConfiguration.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2020 The Android Open Source Project
```

Comment:
Ultra-nit: if this file is new to the repo with this commit, the copyright year should be set to the year the file was created.

Reply:
Thanks. Updated